File: DayOne/Java/LevelOne/PalindromicArray.java
import java.util.Arrays;

public class PalindromicArray{

    public static boolean palindromeCheck(int [] array){

    int newArray [] = new int [array.length];

    int newArrayCount = 0;

    for (int indexCount = array.length - 1; indexCount >= 0; indexCount--) {

        newArray[newArrayCount] = array[indexCount];

        newArrayCount++;
    }
    return Arrays.equals(array,newArray);
      
    }
}

File: DayOne/JavaScript/LevelOne/palindromic-array.js
export const palindromeCheck = (array) =>{

    let newArray = Array(array.length).fill(0);

    let newArrayIndex = 0;

    for (let indexCount = array.length -1; indexCount >= 0; indexCount--){

        newArray[newArrayIndex] = array[indexCount];

        newArrayIndex++;
    }

    return JSON.stringify(array) === JSON.stringify(newArray);

}
